TrailheadApex11/ExercisesPage/src/classes/Ex3PrintingQuotes.cls

```diff
@@ -0,0 +1,21 @@
+/**
+ * Created by Kuba on 29.11.2017.
+ */
+
+public with sharing class Ex3PrintingQuotes implements ExercisesLoader {
+    public String solveTheExercise(List<String> input) {
+        try {
+            String quote = ExercisesController.isInputEmpty(input).get(0);
+            String saidBy = ExercisesController.isInputEmpty(input).get(1);
+            if (quote == null) {
+                return 'You have to type text to be solved';
+            } else {
+                return 'What is the quote? ' + quote
+                        + '\nWho said it? ' + saidBy
+                        + '\n' + saidBy + ' says "' + quote + '"';
+            }
+        } catch (Exception e) {
+            return 'You have to type text to be solved or you didn\'t type required lines of text ';
```

Comment:
Kazałem Ci zrobić nowy tem exceptiona i w sumie to jest spoko miejsce na niego. Różni się ten tekst czasem jednak jak widze, więc po prostu rzucaj go jako wyjątek - z odpowiednim tekstem (który jest przechowywani w staticach RAZ w kodzie ;p)